tests/test_core.py

```diff
@@ -8080,7 +8080,6 @@ def test_pthread_c11_threads(self):
       self.set_setting('INITIAL_MEMORY', '64mb')
     self.do_run_in_out_file_test('tests', 'pthread', 'test_pthread_c11_threads.c')
 
-  @no_asan('flakey errors that must be fixed, https://github.com/emscripten-core/emscripten/issues/12985')
```

Comment:
Can you update the PR title and description including specifically how this change address this issue?

Reply:
Sure no problem. I've updated them. Is my description clear and understandable?